webui/src/components/editfiles.jsx

```diff
@@ -555,7 +558,13 @@ export const FileRecordRow = React.createClass({
                         { this.props.hideFileDownloads ? null :
                             <span className="fileDownloadBadge" style={{marginLeft:'1em', fontSize:11, color: '#888'}}>
                                 <span className="badge" style={{marginLeft:'0.5em'}}> Total Downloads {downloads} </span>
-                            </span> 
+                            </span>
+                        }
+                        {
+                            b2safe_pid ?
```

Comment:
Why not make the condition just `file.b2safe_pid ?`, without defining the `b2safe_pid`.
Also, probably `file.b2safe_pid` should be boolean, not string.

Reply:
Yes I did both these changes